windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*[**,**_privilege-name_]...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available on Windows 7 and above.
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required. For example, the privilege name should be written as "SeAuditPrivilege" but not SE_AUDIT_NAME.
```

Comment:
"is the name of the privilege" -> "is the name of a privilege that the service requires"
"privilegs names" -> "privilege names" (You have an 's' where there should be an 'e')